src/ReleaseHistory.md

```diff
@@ -15,6 +16,9 @@
 * BUGFIX  : Update `merge` command to properly produce runs by tool and version when passed the `--merge-runs` argument. [#2488](https://github.com/microsoft/sarif-sdk/pull/2488)
 * BUGFIX  : Eliminate `IOException` and `DirectoryNotFoundException` exceptions thrown by `merge` command when splitting by rule (due to invalid file characters in rule ids). [#2513](https://github.com/microsoft/sarif-sdk/pull/2513)
 * BUGFIX  : Fix classes inside NotYetAutoGenerated folder missing `virtual` keyword for public methods and properties, by regenerate and manually sync the changes. [#2537](https://github.com/microsoft/sarif-sdk/pull/2537)
+* FEATURE : Provide mechanism to populate `SarifLogger` with a `FileRegionsCache` instance.
```

Comment:
line 19 is a copy of line 30, line 20 is copy of line 31. seems like a merging issue

Reply:
Yes! Thank you for the careful review.